### ISSUE TITLE
Parsing of videos elements

### ISSUE THREAD

**Issue by kulla** [Lodifice/mfnf-pdf-export#29]
Video elements are stored in `<figure>...</figure>` (see https://de.wikibooks.org/wiki/Mathe_f%C3%BCr_Nicht-Freaks:_Satz_vom_Minimum_und_Maximum for an article with videos) and need to be parsed. In the class `MediaWiki2Latex` of `latex.py` they shall be deleted.

**Comment by elbotho** [comment]
Can you guess how many videos we have in the _Analysis I_ book?

**Comment by kulla** [comment]
No, the videos only contain additional explanations but they are not necessary to understand the articles.

**Comment by kulla** [comment]
With https://github.com/Lodifice/mfnf-pdf-export/commit/3f6461b7aeadcf7a6cd2d16116e1962c4a1cb880 there is no need to fix this issue for the first milestone.
